.github/pull_request_template.md

```diff
@@ -0,0 +1,24 @@
+Thank you for your pull request. Please fill the following information.
```

Comment:
Please fill the following information. -> Please provide the following information.